source/releasenotes/2024-12-10-drupal-7-d7es.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Tag1 Extended Support module installed on Drupal 7 upstream
+published_date: "2024-12-05"
+categories: [drupal, action-required]
+---
+
+The default Pantheon Drupal 7 upstream has been updated to include the Tag1 Drupal 7 Extended Support (D7ES) module.
+
+Future LTS updates to both Drupal 7 Core and the D7ES module will be available as drops-7 upstream updates. Learn more in our [Drupal 7 LTS documentation](https://docs.pantheon.io/supported-drupal#d7-core-lts-and-platform-compatibility).
+
+## Action Required
+
+To receive security ongoing security coverage for your contrib modules, apply this upstream update then enable the Tag1 D7ES module.
+
+## Long-Term Support
+
+Pantheon is committed to providing security updates for Drupal 7 for at least two additional years beyond its upcoming end-of-life date on January 5, 2025. This commitment is in partnership with Tag1 Consulting, offering Long-Term Support (LTS) for Drupal 7 at no additional cost.
+
```

Comment:
@scottbuscemi @pwtyler somewhere in here can you add a sentence or two about how our LTS intersects with Autopilot?

Reply:
According to our [documentation](https://docs.pantheon.io/supported-drupal#d7es-usage) "[Autopilot](https://docs.pantheon.io/guides/autopilot) will automatically apply contrib module updates from the Tag1 D7ES service the next time Autopilot runs."

Since it's in our docs, I don't think it's necessary to add to the release note.